node/src/test/resources/default-configuration.toml

```
@@ -84,6 +84,7 @@ auto-propose-enabled = false
 auto-propose-check-interval = "1second"
 auto-propose-max-interval = "1second"
 auto-propose-max-count = 1
+max-block-size = 10
```

```suggestion
max-block-size = 1
```

Just to conform to all the other values.